content/install/amazon.md

```diff
@@ -45,18 +45,20 @@ The autoscaler container can be started with the below command. The container is
 {{< highlight text "linenos=table" >}}
 docker run -d \
   -v /var/lib/autoscaler:/data \
-  -e DRONE_POOL_MIN={DRONE_POOL_MIN} \
-  -e DRONE_POOL_MAX={DRONE_POOL_MAX} \
-  -e DRONE_SERVER_PROTO={DRONE_SERVER_PROTO} \
-  -e DRONE_SERVER_HOST={DRONE_SERVER_HOST} \
-  -e DRONE_SERVER_TOKEN={DRONE_SERVER_TOKEN} \
-  -e DRONE_AGENT_TOKEN={DRONE_AGENT_TOKEN} \
-  -e DRONE_AMAZON_REGION={DRONE_AMAZON_REGION} \
-  -e DRONE_AMAZON_SUBNET_ID={DRONE_AMAZON_SUBNET_ID} \
-  -e DRONE_AMAZON_SECURITY_GROUP={DRONE_AMAZON_SECURITY_GROUP} \
-  -e DRONE_AMAZON_SSHKEY={DRONE_AMAZON_SSHKEY} \
-  -e AWS_ACCESS_KEY_ID={AWS_ACCESS_KEY_ID} \
-  -e AWS_SECRET_ACCESS_KEY={AWS_SECRET_ACCESS_KEY} \
+  -e DRONE_POOL_MIN=${DRONE_POOL_MIN} \
+  -e DRONE_POOL_MAX=${DRONE_POOL_MAX} \
+  -e DRONE_SERVER_PROTO=${DRONE_SERVER_PROTO} \
+  -e DRONE_SERVER_HOST=${DRONE_SERVER_HOST} \
+  -e DRONE_SERVER_TOKEN=${DRONE_SERVER_TOKEN} \
+  -e DRONE_AGENT_TOKEN=${DRONE_AGENT_TOKEN} \
+  -e DRONE_AMAZON_IMAGE=${DRONE_AMAZON_IMAGE} \
```

Comment:
I would prefer we remove.  We want people using our default image.  Specifying an image is optional, and specifying he wrong image (e.g. non-Ubuntu) would result in failure.